config/jest.config.js

```diff
@@ -1,10 +1,11 @@
 const path = require('path');
+process.env.PUPPETEER_CHROMIUM_REVISION = 768783;
```

Comment:
Same here - I think we're introducing some unnecessarily manual processes.

Reply:
I has been removed, same as from package.json